CHANGES_NEXT_RELEASE

```diff
@@ -6,3 +6,4 @@
 - FIX Correctly return 404 when asking to remove a not existing device (#585)
 - Add: support HTTPS requests toward Keystone (if url is used in the authorization configuration)
 - Add: support OAuth2 provider as authentication backend (#591)
+- Fix: incomplete HTTPS support for subscriptions (#593)
```

Comment:
Small adjustment, pls:

Fix: incomplete HTTPS support for subscriptions (#593)

to

Fix: incomplete HTTPS support for NGSIv1 subscriptions (#593)

Reply:
fix by e2ab72a